### DISCUSSION TITLE
Should we adopt Tux as a mascot?

### DISCUSSION THREAD

**Discussion by Pythone** [kantord/LibreLingo#2610] [Ideas]
Tux is a mascot in multiple apps and games. Can we have Tux as a mascot in this app?

**Comment by kantord** [comment]
LibreLingo already has a mascot called Lily, who is a turtle.

Tux is closely related to Linux, whereas LibreLingo doesn't have much in common with Linux except for the fact that both Linux and LibreLingo are free software :thinking: 

So I think adopting Tux as a mascot would be confusing, and besides we already have a mascot that I think represents the project better.

**Comment by cutthroat78** [comment]
At some point, we could have each challenge (that doesn't have an image) have a character with it to make it look like the character is saying it. Like what, Duolingo has. If we do this we could have Lily as well as Tux and other FOSS characters.

If we do add a feature like this I would like it to be something you can enable or disable, so that people can customise their learning experience.

**Reply by kantord** [reply]
> At some point, we could have each challenge (that doesn't have an image) have a character with it to make it look like the character is saying it. Like what, Duolingo has. If we do this we could have Lily as well as Tux and other FOSS characters.
> 
> If we do add a feature like this I would like it to be something you can enable or disable, so that people can customise their learning experience.

That is a good idea
